commands/animals/cat.js

```diff
@@ -23,8 +24,7 @@ class Cat extends Social {
       }
 
       const msg = await message.channel.send(`<a:typing:397490442469376001> **${message.member.displayName}** is petting a cat...`);
-      const { body } = await snekfetch.get("https://api.weeb.sh/images/random?type=animal_cat")
-        .set("Authorization", `Wolke ${process.env.WEEBSH}`)
+      const { body } = await snekfetch.get("https://nekos.life/api/v2/img/meow")
         .set("User-Agent", `Misaki/${version}/${this.client.user.id === "396323622953680910" ? "Production" : "Development"}`);
```

Comment:
You forgot to remove this line since you're not using `weeb.sh` anymore

Reply:
Ok